lib/system.nim

```diff
@@ -4534,9 +4534,9 @@ when defined(genode):
 import system/widestrs
 export widestrs
 
-when defined(ioverride):
-  include "$projectpath/iooverride"
-  export ioverride as io
+when defined(iooverride):
+  include "$projectpath/iooverride" as io
```

Comment:
What does `include ... as even mean`? I don't think this is valid Nim.

Reply:
Ups, just copied from fatal.nim. It should be an import